docs/content/fin/fin-help.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "All commands reference"
+title: "fin help"
```

Comment:
Keep the title "All commands reference"

Reply:
It is the title that you set in help generation script. I manually changed it in another commit but the script replaced it again.

Reply:
Right. Let's then leave it as set in `scripts/fin-help-generate.sh` - "fin help"